scripts/trafficlight.js

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const cypress = require('cypress');
+const process = require('process');
+const crypto = require('crypto');
+const http = require('http');
+
+function setup(trafficlightUrl, uuid) {
+    console.log("Setting up trafficlight client");
+
+    const data = JSON.stringify({
+        type: "element-web",
+        version: "0.15.0",
```

Comment:
which version is this for?

Reply:
Ah yes, good question: how would I best programatically get the current version of element web (which i believe is different to that of matrix-react-sdk, but is what's under test) in this context? I could make the http call out to the element web under test (/version) i suppose?

Reply:
Depends how you are running the Element under test, running `yarn start` results in:

![image](https://user-images.githubusercontent.com/2403652/183681475-c07ed7c5-676f-4744-bc5e-ba0be913f18f.png)

Reply:
As we don't use this version information for deciding which client to use for a given test (at the moment, but it will need to later) i'm going to make it say "unknown" for now, as it's not essential for this PR's function.

It might be good if UNSET could be more like the synapse version, which is something like `<branch-name>-<git sha prefix>` or something, when built from git. Though presumably you need the react-sdk version in there as well if you want to truely identify it.

Reply:
We do that if you run a package, but in-development it doesn't do that because its non-trivial to do in webpack-dev-server